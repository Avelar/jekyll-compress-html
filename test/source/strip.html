---
layout: compress
---

<p>One</p><p>Two</p>


