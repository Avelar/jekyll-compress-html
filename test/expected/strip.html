<p>One</p><p>Two</p>
